app/main.py

```diff
@@ -21,18 +22,17 @@ def create_tables(engine, psql_environment):
     if 'sqlite' in str(engine.url) and psql_environment:
         raise models.PSQLEnvironmentError(
             "You're trying to use PSQL features on SQLite env.\n"
-            "Please set app.config.PSQL_ENVIRONMENT to False "
-            "and run the app again."
-        )
+            'Please set app.config.PSQL_ENVIRONMENT to False '
+            'and run the app again.')
```

Comment:
The entire website uses double quotes, unless you mean to go and edit every page to confirm to a single quote style, this change isn't really helpful.

Reply:
done